lib/exportOverlays.nix

```
@@ -60,7 +60,8 @@ let
       fakePkgs = {
         callPackage = it: it;
         isFakePkgs = true; # Overlay maintainers may throw an exception in case this key is set in case overlay is not compatible with this function.
-        lib = flake-utils-plus.lib.internal;
+        # import the lib input from first channel
+        lib =  if (builtins.attrNames channels) == [] then flake-utils-plus.lib.internal else inputs."${(builtins.elemAt (builtins.attrNames channels) 0)}".lib;
```

How do we know what is first input? Also cant we simplify this as such:
```suggestion
        lib =  if (builtins.attrNames channels) == [] then flake-utils-plus.lib.internal else (builtins.elemAt (builtins.attrValues channels) 0).lib;
```

---

I don't know what you mean. IMO,  if we use an actual channel that should replace the flakePkg to https://github.com/divnix/digga/pull/483/files#diff-50b278a919472d6eab9c50638883df55c1129fb60f71ded49b06673428be8ceeR49